requirements.txt

```
@@ -1,6 +1,6 @@
 dwave-ocean-sdk>=5.3.0
-scikit-learn>=1.1.1
+scikit-learn>=1.6.1
 openml==0.15.1
-pandas>=1.4.2
-dash==2.7.0
-dwave-scikit-learn-plugin==0.1.0
\ No newline at end of file
+pandas>=2.3.1
+dash==3.2.0
+dwave-scikit-learn-plugin==0.1.0
```

Changes now released in latest version.
```suggestion
dwave-scikit-learn-plugin==0.2.0
```